src/extension.ts

```diff
@@ -175,6 +176,12 @@ export function activate(context: vscode.ExtensionContext) {
             return edgeManager.initializeSample(name, url, platform, outputChannel);
         });
 
+    initCommandAsync(context, outputChannel,
+        "azure-iot-edge.showMarketplace",
```

Comment:
seems the importModule method will not add module into deployment manifest right? it only returns module info? How about we have a command "showMarketplace" that only shows the marketplace page. And the action in the page will trigger the import module process. Besides, the integeration in add module could just show the market place page.

Reply:
Oh, I need to remove this. Good catch. `showMarketplace` is useless.